daffodil-japi/src/main/scala/org/apache/daffodil/japi/packageprivate/Utils.scala

```diff
@@ -42,6 +44,17 @@ private[japi] object ValidationConversions {
   }
 }
 
+private[japi] object XMLOutputStyleConversions {
+
+  def modeToScala(mode: XMLOutputStyle): SXMLOutputStyle.Type = {
```

Comment:
`mode` doesn't make sense in this context. We use mode in ValidationConversions because it is converting a `ValidationMode`. In this case, if you want a short name, maybe `style` is more appropriate?